core-document-store/pom.xml

```diff
@@ -48,6 +48,11 @@
 			<artifactId>oxcore-util</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Can you try to use gluu-orm-annotation instead of this artifact to get rid of specific DB dependency?